packages/perseus/src/widgets/interactive-graphs/interactive-graph.testdata.ts

```diff
@@ -29,6 +35,8 @@ export const angleQuestion: PerseusRenderer = interactiveGraphQuestionBuilder()
     })
     .build();
 
+export const angleItem: PerseusItem = createPerseusItem(angleQuestion);
```

Comment:
Same here, we don't need to be redeclaring a bunch of test data when we could just be wrapping the existing test data in `createPerseusItem`